### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by yufeng9819** [luost26/diffusion-point-cloud#20]
Hi！ I am interested in your wonderful work!

I have a question about training. In your training file "train_gen.py", you set the training iteration number to 'inf' and the training loss is around 60.5, also the grad is around 0.2 to 0.3. So I want to know when to stop straining and I cannot judge stop time just by the loss and grad. Also I want to know why the training loss almost ever decrease a lot. 

Looking forward to your response! 

Best.

**Comment by luost26** [comment]
Hi,

Thank you for your interest in our project. Could you take a look at the generated point clouds in the tensorboard? Sometimes the absolute value of loss does not tell anything.

**Comment by yufeng9819** [comment]
I got it! Thanks for  your response
